cookiecutter.yaml

```
@@ -12,4 +12,7 @@ default_context:
     hosting_web_app_address: "https://OpenMontana.github.io/montana-legislature-council-data-project"
     firestore_region: "us-west1"
     event_gather_timedelta_lookback_days: 10
+    # TODO check with mark on new event gather schedule
     event_gather_cron: "26 0,6,12,18 * * *"
```

```suggestion
    event_gather_cron: "26 0 * * *"
```

@mfrischmuth I suggest we change this to run nightly, insead of every 6 hours.

---

`26 0 * * *` roughly translates (from cron to english) as "At 00:26."